targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/TARGET_ADAFRUIT_NRF52840_FEATHER/README.md

```diff
@@ -0,0 +1,143 @@
+# Adafruit Feather nRF52840 Express
```

Comment:
We probably shouldn't include this file in this PR - it looks like a general tutorial on how to work with Mbed OS, which we have official documentation for https://os.mbed.com/docs/mbed-os/v6.3/introduction/index.html

The only kind of READMEs under `targets` are technical details related to the _porting_ of the targets.